inttest/kubectl/kubectl_test.go

```diff
@@ -68,23 +68,23 @@ func (s *KubectlSuite) TestEmbeddedKubectl() {
 			Check: func(output string, e error) {
 				s.Require().NoError(e)
 				// Check for debug log messages
-				s.Require().True(strings.Contains(output, "round_trippers.go"))
+				s.Require().True(strings.Contains(output, "round_trippers.go"), "Unexpected output: %v", output)
 			},
 		},
 		{
 			Name:    "full command plugin loader",
 			Command: "k0s kubectl foo",
 			Check: func(output string, e error) {
 				s.Require().NoError(e)
-				s.Require().Equal(output, "foo-plugin")
+				s.Require().Equal(output, "foo-plugin", "Unexpected output: %v", output)
```

Comment:
Shouldn't this be

```suggestion
				s.Require().Equal("foo-plugin", output)
```

? The expected value comes first IIRC. Testify should also show a quite readable diff in case the values are not equal.

Same on lines 87, 113.